src/helpers/userHelper.js

```diff
@@ -1208,123 +1237,129 @@ const userHelper = function () {
         });
       });
   };
-
-  // 'Lead a team of X+'
+  // 'Lead a team of x+'
   const checkLeadTeamOfXplus = async function (
     personId,
     user,
-    badgeCollection,
+    badgeCollection
   ) {
-    if (!hasPermission(user.role, 'checkLeadTeamOfXplus')) {
-      return;
-    }
+    const leaderRoles = [
+      "Mentor",
+      "Manager",
+      "Administrator",
+      "Owner",
+      "Core Team",
+    ];
+    const approvedRoles = ["Mentor", "Manager"];
+    console.log(user.role);
+    if (!approvedRoles.includes(user.role)) return;
     let teamMembers;
     await getTeamMembers({
       _id: personId,
     }).then((results) => {
       if (results) {
-        teamMembers = results.myTeam;
+        teamMembers = results.myteam;
```

Comment:
It is myteam inside the database. I'd love to open a pr down the road to fix that but I want to make sure that the function works properly before I change anything else in mongo to avoid unforeseen issues arising.